vms/avm/service.md

```diff
@@ -783,6 +783,47 @@ curl -X POST --data '{
 }
 ```
 
+### `info.getTxFee`
+
+Get the fees of the network.
+
+**Signature**:
+
+```
+info.getTxFee() ->
+{
+  txFee: uint64,
+  createAssetTxFee: uint64,
+}
+```
+
+- `txFee` is the default fee for making transactions.
+- `createAssetTxFee` is the fee for creating a new asset.
+
+All fees are denominated in nAVAX.
+
+**Example Call**:
+
+```sh
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     : 1,
+    "method" :"avm.getTxFee",
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/X
+```
+
+**Example Response**:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "txFee": "1000000",
+    "createAssetTxFee": "10000000"
```

Comment:
Currently the service returns `uint64` and not the `json.Uint64` helper - so I don't think these are quoted in the response... Right? I think either this doc should be updated or we should be returning quoted strings.

In the past we returned quoted numbers to avoid floating point inaccuracies in javascript (because all numbers in js are floating point and they lose precision after 2^53).